docs/DISCORD_FAQ.md

```diff
@@ -24,8 +24,10 @@ _Frequently Asked Questions_
 > **7: My Jacob Contest Display crops are wrong, how do I fix this?**
 > Run the command `/shclearcontestdata` to clear the Jacob contest data.
 
-> **8: How can I get bigger crop hit boxes?**
-> Use [Patcher](<https://sk1er.club/mods/patcher>) to have 1.12 hitboxes in 1.8.9.
+> **8: How can I get bigger crop hitboxes?**
+> Use Patcher or PolyPatcher to have 1.12 hitboxes in 1.8.9.
+> - [Polyfrost's PolyPatcher](<https://modrinth.com/mod/patcher>) (a fork of Patcher with more features and performance improvements)
```

Comment:
Maybe it's fairer to describe the difference as "with slightly different features and bug fixes"? Not sure how much the newer version increases performance. Also since poly patcher has some features removed (e.g. block breaking particles) the wording "with more features" is inaccurate.

Maybe include that one version is build with essentials, the other with oneconfig? Idk 

Also not sure what mod to show first in the list. The more known origins of the updated mod. Any good arguments for either? (Also asking other contributors for their comment here)